### DISCUSSION TITLE
The official documentation and the github page

### DISCUSSION THREAD

**Discussion by jerome-diver** [chakra-ui/chakra-ui#9257] [Ideas]
### Description

When i read the doc to install and use the chakra-ui (the last) with Vite, i expected it to work as indicated.
It does not.
After to loose a lot of time on error as not existing next-themes from @/components/ui/provider fake?old indication on your documentation, i find on github readme page the correct solution to use (then has to remove the import of Provider from wrong @/components/ui/provider.
Please, can you control seriously your documentation and update them both in the same time so that we can hope to understand something and go ahead the story with this lib ?
Thank you very much for that high level careful attention.

### Link to Reproduction

no need

### Steps to reproduce

1. install react with vite (typescript with babel)
2. open the official documentation and trust about lastest v3 chakra-ui
3. go to vite indication
4. loose ytour time around
5. go to github page to fix the wrong official documentation

### Chakra UI Version

latest this day i publish

### Browser

nevermind

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

### Additional Information

thank you for your documentation job to follow the reality of the code (maybe slow down the code to update first the doc).

**Comment by TinsFox** [comment]
I'm also having the issue of agreeing, react-icons are missing in addition to next-theme. I want to submit a PR to fix this, but there is a problem that the icon library used in the vite example is lucide-react. This makes me wonder which icon library chakra-ui uses.

@segunadebayo Can you answer my questions? I'd be happy to open a PR to fix this.

**Comment by jerome-diver** [comment]
I can not understand why they put this in "discussion" instead of issue. 
Does someone can explain to me ? It looks like a game to not show the problem but to put under the carpet... so i can imagine that the issues list doesn't reflect the real issues list that was never resolved. How many there is under the carpet ?
